hack/images/titus-sshd/run-titus-sshd

```diff
@@ -0,0 +1,55 @@
+#!/titus/sshd/bin/busybox sh
```

Comment:
Maybe shell check this file?

Reply:
I'll get this on the next PR and see if I can grow it to include any script.
I think I'll have to adjust the builder docker image to get shellcheck in it